### PR TITLE
Inspection_oM: Added two new properties to Issues

### LINKED ISSUE #1167: Inspection_oM: Add new property "DateClosed" & "AuditID" to Issues
#### Description:
Add new property `DateClosed` to Issues. Initial value does not not need to be read from IAuditor pull. Value can default to 00 : 00 : 00 to be populated at later date from 3D repo or other. 
Add new property `AuditID`. Required for loop around and overwriting Mongo Issues edited in excel/3Drepo.

### PR DESCRIPTION
**Pull request by BSochacki**
### Issues addressed by this PR
Closes #1167 

### Test files
Files are not required for testing. To test, create issue object and check the new properties `DateClosed` and `AuditID` are present

### Changelog
Added 2 new properties to Issue object.

### CODE REVIEW

**Comment by enarhi** [comment]
@BHoMBot check compliance

**Review by enarhi — DISMISSED** [review]
LGTM

**Comment by enarhi** [comment]
@BHoMBot check compliance

**Review by enarhi — APPROVED** [review]
Updating review per addition of IssueNumbers, LGTM.

File: Inspection_oM/Audit.cs [hunk 1]
```diff
@@ -91,8 +91,8 @@ public class Audit : BHoMObject
         [Description("Installation progress objects from the audit (Each of these corresponds to the status of a specific area inspected)")]
         public virtual List<InstallationProgress> InstallationProgressObjects { get; set; } = new List<InstallationProgress>();
 
-        [Description("Issues from audit. These are issues identified during the audit for various areas that need to be addressed, including supporting images of the work as applicable.")]
-        public virtual List<Issue> Issues { get; set; } = new List<Issue>();
+        [Description("Issues from audit. These are reference IssueNumbers to corresponding issues identified during the audit for various areas that need to be addressed, which can include supporting images of the work as applicable.")]
+        public virtual List<string> IssueNumbers { get; set; } = new List<string>();
```
Comment (FraserGreenroyd) [inline Inspection_oM/Audit.cs:95]: This property will require versioning @enarhi - I'm happy to merge if you accept to doing this before 4.1 beta? It'll need doing in the Versioning_Toolkit so not able to be done as part of this PR unfortunately.

Comment (enarhi) [reply Inspection_oM/Audit.cs:95]: Will follow up with a pr in Versioning_Toolkit accordingly in the next few days.

**Pull request merged**